docs/api.adoc

```diff
@@ -467,13 +390,17 @@ Show a contract that was hidden before by the command `emerald_hideContract`.
 <-- {"jsonrpc": "2.0", "error": {"code": -32000, "message": "Contract doesn't exist"}, "id": "1"}
 ----
 
+=== emerald_updateContract
+
+TBD
+
 === emerald_importContract
 
-Add a new contract and store the Application Binary Interface (ABI) locally.
+Import a new smart contract Application Binary Interface (ABI) into local storage.
```

Comment:
"Local storage" typically refers to web browser storage, so I prefer "locally", just like with `emerald_importAccount`